ios/nskeyedarchiver/objectivec_classes.go

```diff
@@ -117,6 +124,62 @@ func NewXCTestConfiguration(
 	return XCTestConfiguration{contents}
 }
 
+func createTestIdentifierArray(productModuleName string, tests []string) interface{} {
```

Comment:
lets return a `NSMutableArray` from here and make the nil check where you assign the value to `contents` (same for the `createTestsSet`)

Reply:
Done.